app/components/userFeedback/FeedbackFormBox.tsx

```diff
@@ -37,15 +38,24 @@ export type FeedbackBoxProps = {
   readonly destination: string;
   readonly shouldFocus: boolean;
   readonly onSubmit: () => void;
+  readonly feedback?: FeedbackType;
+};
+
+const getWasHelpfulFromUrl = (url: string) => {
+  const params = new URLSearchParams(url);
+  const value = params.get("wasHelpful");
+  return value === "yes" || value === "no" ? value : null;
```

Comment:
if we're just testing for _any_ non-falsy value, could we do something like 
```suggestion
  return value ?? null;
```